packages/standard-minifier-css/plugin/minify-css.js

```diff
@@ -119,26 +119,73 @@ var mergeCss = Profile("mergeCss", function (css) {
 
   var newMap;
 
+  // Compose the concatenated file's source map with source maps from the
+  // previous build step if necessary.
   Profile.time("composing source maps", function () {
-    // If any input files had source maps, apply them.
-    // Ex.: less -> css source map should be composed with css -> css source map
-    newMap = sourcemap.SourceMapGenerator.fromSourceMap(
-      new sourcemap.SourceMapConsumer(stringifiedCss.map));
+    var concatConsumer;
 
-    Object.keys(originals).forEach(function (name) {
+    newMap = new sourcemap.SourceMapGenerator();
+    concatConsumer = new sourcemap.SourceMapConsumer(stringifiedCss.map);
+
+    // Create a dictionary of source map consumers for fast access
+    var consumers = Object.keys(originals).reduce(function (consumers, name) {
```

Comment:
I may be biased against `Array.prototype.reduce`, but I think this might be clearer if you defined `var consumers = Object.create(null)` above, and simply modified it in a `.forEach` loop.